zigpy/application.py

```diff
@@ -539,6 +540,9 @@ async def register_endpoints(self):
             )
         )
 
+        for add_endpoint in self.config.get(CONF_ADDITIONAL_ENDPOINTS):
```

Comment:
Once the endpoint dictionary is parsed in `zigpy/config/__init__.py`, this code can just become:

```Python
for endpoint in self.config[conf.CONF_ADDITIONAL_ENDPOINTS]:
    await self.add_endpoint(endpoint)
```